_includes/charts/tigera-operator/templates/crds/01-crd-installation.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: installations.operator.tigera.io
+  annotations:
```

Comment:
Has this been added to the operator (and the changes below)? We update this CRD with the one that gets generated from the operator when needed.

Reply:
It sure has! https://github.com/tigera/operator/commit/3ddcbc5965019322f20bb771e54334b492bd57d7

Reply:
That change has the changes below (to the description and enum). I was also concerned with this annotation too which I don't see in that PR and looking at master doesn't exist.